IntercomFirmwareTool.Core/Payload/mqtt/README.md

```
@@ -122,6 +122,39 @@ re-run the tool for) is out of scope; the reservation above avoids all of it.
+
+```sh
+iptables() { command iptables -w "$@"; }
+```
+
+Shadowing `iptables` with a function that injects `--wait` makes **every** factory
+call **block** for the xtables lock instead of dropping a rule (`command` reaches
+the real binary, so there is no recursion). The insert is **idempotent** and
+preserves the script's mode/owner. "Already hardened" is judged by the operative
+**function line** — an active (uncommented) `iptables() { … -w … }` positioned
+**before** the first factory `iptables` call — not by the `IntercomFirmwareTool
+#145` marker comment (which is only a human-readable delimiter): a script whose
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)